.trunk/trunk.yaml

```diff
@@ -36,7 +36,7 @@ lint:
     - linters:
         - ALL
       paths:
-        - linters/**/test/*
+        - linters/**/test*/*
```

Comment:
Allows for more lenience (e.g. `test` vs. `tests`)

Reply:
__snapshots__ is autogenerated in the original package, but when specifying a path in `jest-specific-snapshot`, you have to specify a destination, so I stuck to the convention.